packages/pyodide-kernel-extension/package.json

```diff
@@ -31,19 +31,20 @@
     "schema/*.json"
   ],
   "scripts": {
-    "build": "jlpm build:lib && jlpm build:labextension:dev",
-    "build:prod": "jlpm build:lib && jlpm build:labextension",
+    "build": "jlpm build:lib && jlpm build:labextension:dev && jlpm dev",
```

Comment:
Not sure we need to run the `jupyter labextension develop` command everytime we `jlpm run build`. Once at the top-level should be enough.

Reply:
I added it because of the `clean` before a `build:prod` in quickstart, as without something checking any of these states (or failing if they do) it's pretty easy to get to a where stuff gets out of sync.

I'll see if it can move back to `quickstart`... usually end up having to run the whole thing anyway.

Reply:
0238adc